src/lib.rs

```diff
@@ -835,11 +820,26 @@ pub type TextureViewDescriptor<'a> = wgt::TextureViewDescriptor<Option<&'a str>>
 pub type SamplerDescriptor<'a> = wgt::SamplerDescriptor<Option<&'a str>>;
 
 /// A swap chain image that can be rendered to.
-pub struct SwapChainOutput {
+pub struct SwapChainTexture {
     pub view: TextureView,
     detail: <C as Context>::SwapChainOutputDetail,
 }
 
+/// The result of a successful call to `SwapChain::get_next_frame`.
+pub struct SwapChainFrame {
+    pub output: SwapChainTexture,
+    pub suboptimal: bool,
```

Comment:
I know I'm too late to the party, but can we please copy this documentation text from here about what "suboptimal" means in there?
https://github.com/gfx-rs/gfx/blob/bbfdebe6135a9161500594443be8aae1f802a8cc/src/hal/src/window.rs#L456
Saw this now and got confused & curious so I followed the trail to find the doc, now it makes sense. But being relatively new to gfx-rs/ that was long trip ;)

(though I'd argue the name is a bit off, nothing that can't be solved with docs though)

Reply:
that would make sense! We need more documentation in general

Reply:
Ah that's encouraging :) - gonna do a pr once I have at least one more doc thing to make it worthwhile.
Overall I think the in-library docs are getting decent already. Manual like docs & samples is ofc much harder.